IPython/terminal/debugger.py

```diff
@@ -7,6 +7,7 @@
 from IPython.core.completer import IPCompleter
 from .ptutils import IPythonPTCompleter
 from .shortcuts import create_ipython_shortcuts
+from . import embed
```

Comment:
For consistency:

```suggestion
from .terminal.embed import embed
```

Reply:
That's not the same import, I can change it to one of those if that's prefered:
```
from IPython.terminal import embed
from IPython import embed
import IPython.terminal.embed as embed
```

I can't import only the class I need, which I'd normally do, I think because it creates a circular import from ``IPython/__init__.py``.

Reply:
Actually the circular import is embed -> interactiveshell -> debugger -> ember.

Reply:
Oh, wait, you're right, brainfart on my part.

Reply:
@iwanb Brainfart on my part - please disregard :).